### DISCUSSION TITLE
Feature Request: More Push Notifications for daily workflow

### DISCUSSION THREAD

**Discussion by Hona** [community/community#111273] [Mobile]
### Select Topic Area

Product Feedback

### Body

**Hi GitHub Mobile Team,**

I love using the app to supplement my daily workflow & keep up to date with request reviews.

There are a few other types of push notifications I'd like to be able to enable/configure.

- Comment added to subscribed thread (my PR, issue discussion)
- New Review added to my PR (feedback, approved, commented)
- Subscribed PR was merged (e.g. someone merged my PR)

I currently refresh the GitHub Notifications page on desktop every now and then (akin to doom scrolling). Would be awesome to just get notified :)

**Comment by esahmed21** [comment]
Thank you for getting in touch, @Hona ! I appreciate your input and will certainly bring your suggestion to the team for further consideration and discussion.

**Comment by Sheena131** [comment]
Access to the public
